src/configuration_wrapper/configuration_wrapper.rs

```diff
@@ -9,7 +10,7 @@ impl ConfigurationWrapper {
     pub fn new() -> Result<Self, ConfigError> {
         let env_name = std::env::var("ENV_NAME").unwrap_or_else(|_| "development".into());
 
-        println!("Loading configuration for {} environment... ", env_name);
+        info!("Loading configuration for {} environment... ", env_name);
```

Comment:
Nice, I didn't knew that you could call log like this before the ConfigurationWrapper finished instancialize itselfs ! Great find